lib/Translation/Translator.php

```diff
@@ -366,7 +361,7 @@ protected function checkForEmptyTranslation($id, $translated, $parameters, $doma
             }
         }
 
-        return $translated;
+        return !empty($translated) ? $translated : $id;
```

Comment:
Was this change done by accident? It is now impossible to know if something was translated or not. For example:

Let's say I call the method checkForEmptyTranslation() with string $id "taxi" and target language "tr" (Turkish). 

If I get back "taksi" I'll know the word was translated because it is significantly different. However if no translation exists I will still get back "taxi" and I'll have no idea if the translation existed or not.